Formula/grafana.rb

```diff
@@ -0,0 +1,149 @@
+require "language/node"
+
+class Grafana < Formula
+  desc "Gorgeous metric visualizations and dashboards for timeseries databases."
+  homepage "http://grafana.org"
+  url "https://github.com/grafana/grafana/archive/v3.0.4.tar.gz"
+  sha256 "f26a374326e64a8f83c57fdf916ba1c8524dd55002dfe628b4854d05fed3715a"
+
+  head "https://github.com/grafana/grafana.git"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    grafana_path = buildpath/"src/github.com/grafana/grafana"
+    grafana_path.install Dir["*"]
+    grafana_path.install ".jscs.json", ".jsfmtrc", ".jshintrc", ".bowerrc"
+
+    cd grafana_path do
+      inreplace "package.json", '"sass-lint": "^1.6.0"', '"sass-lint": "1.7.0"'
+      system "go", "run", "build.go", "setup"
```

Comment:
(It does download external go libs, of course, but those are all vendored with godeps.)